content/docs/securitypolicy/_index.md

```diff
@@ -6,9 +6,20 @@ Description: >
   Dell Container Storage Modules (CSM) Security Policy
 ---
 
+# Security Policy
 
-The CSM services/repositories are inspected for security vulnerabilities via [gosec](https://github.com/securego/gosec), 
-Instructions for reporting a vulnerability can be found on the [Dell Vulnerability Policy](https://www.dell.com/support/contents/en-in/article/product-support/self-support-knowledgebase/security-antivirus/alerts-vulnerabilities/dell-vulnerability-response-policy#:~:text=To%20report%20a%20security%20vulnerability%20or%20issue%20in%20Dell.com,instructions%20to%20reproduce%20the%20issue)
+The CSM services/repositories are inspected for security vulnerabilities via [gosec](https://github.com/securego/gosec).
 
+Every issue detected by `gosec` is mapped to a [CWE (Common Weakness Enumeration)](http://cwe.mitre.org/data/index.html) which describes in more generic terms the vulnerability.  The exact mapping can be found at https://github.com/securego/gosec in the `issue.go` file. The list of rules checked by `gosec` can be found [here](https://github.com/securego/gosec#available-rules).
 
-CSM recommends to stay on the [lastest release](https://github.com/dell/csm/releases/latest) of Dell Container Storage Modules to take advantage of new features, enhancements, bug fixes, and security fixes.
+In addition to this, various security checks get executed against a branch when a pull request is created/updated.  Please refer to [pull request](/docs/CONTRIBUTING.md#pull-requests) for more information.
+
+# Reporting Security Issues/Vulnerability
```

Comment:
Typo - Vulnerability should be Vulnerabilities

Reply:
done